main.js

```diff
@@ -40,3 +41,37 @@ function createMovie(movie) {
   cardCover.style.backgroundImage = `url(${movie.image})`;
   return temp;
 }
+
+const btnsOpenModal = document.querySelectorAll(".movieTemplate__button");
+const btnCloseModal = document.querySelector(".modal__close-modal");
+const modal = document.querySelector(".modal");
+const overlay = document.querySelector(".overlay");
+const cancelBtn = document.querySelector(".modal__btns__cancel");
+const btnBook = document.querySelector(".modal__btns__book");
+
+const openModal = function () {
+  modal.classList.remove("hidden");
+  overlay.classList.remove("hidden");
+};
+
+const closeModal = function () {
+  modal.classList.add("hidden");
+  overlay.classList.add("hidden");
+  document.querySelector(".modal__offLine").style.color = "#464646";
+};
+
+btnCloseModal.addEventListener("click", closeModal);
+overlay.addEventListener("click", closeModal);
+
+
+const offlineMessage = function () {
+  console.log("Offline message function is running");
+  document.querySelector(".modal__offLine").style.color = "#800020";
+};
+
+btnBook.addEventListener("click", offlineMessage);
+
+cancelBtn.addEventListener("click", closeModal);
+
+for (let i = 0; i < btnsOpenModal.length; i++)
+  btnsOpenModal[i].addEventListener("click", openModal);
```

Comment:
I think that there will be an error when visiting aboutUs.html becouse the elements of modal does not exist there. Try to make som kind of conditional that checks the url of the webpage and make it so your added JS code doesnt run if page url includes aboutUs. 

Also there seems to be some merge conflicts that needs to be solved. 

Other than that i think the code and styling looks good. Nice job with the nesting in modal.scss, very well organized and easy to understand.